tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartLocaleGrowth.java

```diff
@@ -0,0 +1,347 @@
+        } catch (RuntimeException e1) {
+            throw e1;
+        }
+        Map<String, FoundAndTotal> data = new HashMap<>();
+        char c = 0;
+        Set<String> latestAvailable = newFactory.getAvailableLanguages();
+        for (String locale : newFactory.getAvailableLanguages()) {
+            if (!matcher.reset(locale).matches()) {
+                continue;
+            }
+            if (!latestAvailable.contains(locale)) {
+                continue;
+            }
+            if (SUPPLEMENTAL_DATA_INFO.getDefaultContentLocales().contains(locale)
+                || locale.equals(LocaleNames.ROOT)
+                || locale.equals(LocaleNames.UND)
+                || locale.equals("supplementalData")) {
+                continue;
+            }
+            char nc = locale.charAt(0);
+            if (nc != c) {
+                System.out.println("\t" + locale);
+                c = nc;
+            }
+            if (DEBUG_FILTER != 0 && DEBUG_FILTER != nc) {
+                continue;
+            }
+            CLDRFile latestFile = null;
+            try {
+                latestFile = latestFactory.make(locale, true);
+            } catch (Exception e2) {
+                System.out.println("Can't make latest CLDRFile for: " + locale + "\tpast: " + mainDir + "\tlatest: " + Arrays.asList(latestFactory.getSourceDirectories()));
+                continue;
+            }
+            CLDRFile file = null;
+            try {
+                file = newFactory.make(locale, true);
+            } catch (Exception e2) {
+                System.out.println("Can't make CLDRFile for: " + locale + "\tpast: " + mainDir);
+                continue;
+            }
+            // HACK check bogus
+//            Collection<String> extra = file.getExtraPaths();
+//
+//            final Iterable<String> fullIterable = file.fullIterable();
+//            for (String path : fullIterable) {
+//                if (path.contains("\"one[@")) {
+//                    boolean inside = extra.contains(path);
+//                    Status status = new Status();
+//                    String loc = file.getSourceLocaleID(path, status );
+//                    int debug = 0;
+//                }
+//            }
+            // END HACK
+            Counter<Level> foundCounter = new Counter<>();
+            Counter<Level> unconfirmedCounter = new Counter<>();
+            Counter<Level> missingCounter = new Counter<>();
+            Set<String> unconfirmedPaths = null;
+            Relation<MissingStatus, String> missingPaths = null;
+            unconfirmedPaths = new LinkedHashSet<>();
+            missingPaths = Relation.of(new LinkedHashMap<MissingStatus, Set<String>>(), LinkedHashSet.class);
+            VettingViewer.getStatus(latestFile.fullIterable(), file,
+                pathHeaderFactory, foundCounter, unconfirmedCounter,
+                missingCounter, missingPaths, unconfirmedPaths);
+
+            // HACK
```

Comment:
i think `HACK` means "don't count these against coverage" but more wording would be helpful here.